Views/Home/Privacy.cshtml

```diff
@@ -1,6 +1,110 @@
+<h2 class="adsimple-311192666">BootstrapCDN Datenschutzerklärung</h2>
+<p>Um Ihnen all unsere einzelnen Webseiten (Unterseiten unserer Website) auf allen Geräten schnell und sicher ausliefern zu können, nutzen wir das Content Delivery Network (CDN) BootstrapCDN des amerikanischen Software-Unternehmens StackPath, LLC 2012 McKinney Ave. Suite 1100, Dallas, TX 75201, USA.</p>
+<p>Ein Content Delivery Network (CDN) ist ein Netzwerk regional verteilter Server, die über das Internet miteinander verbunden sind. Durch dieses Netzwerk können Inhalte, speziell sehr große Dateien, auch bei großen Lastspitzen schnell ausgeliefert werden.</p>
+<p>BootstrapCDN funktioniert so, dass sogenannte JavaScript-Bibliotheken an Ihren Browser ausgeliefert werden. Lädt nun Ihr Browser eine Datei vom BootstrapCDN herunter, wird Ihre IP-Adresse während der Verbindung zum Bootstrap-CDN-Server an das Unternehmen StockPath übermittelt.</p>
+<p>StackPath erwähnt auch in der hauseigenen Datenschutzerklärung, dass das Unternehmen aggregierte und anonymisierte Daten von diversen Diensten (wie BootstrapCDN) für die Erweiterung der Sicherung und für andere StackPath-Dienste und Clients verwenden. All diese Daten können aber keine Person identifizieren.</p>
+<p>Wenn Sie diese Datenübertragung unterbinden wollen, können Sie einen JavaScript-Blocker (siehe beispielsweise <a class="adsimple-311192666" href="https://noscript.net/" target="_blank" rel="noopener noreferrer">https://noscript.net/</a>) installieren oder in Ihrem Browser die Ausführung von JavaScript-Codes deaktivieren. Bitte beachten Sie aber, dass dadurch die Website nicht mehr das gewohnte Service (wie etwa schnelle Ladegeschwindigkeit) bieten kann.</p>
+<p>StackPath ist aktiver Teilnehmer beim EU-U.S. Privacy Shield Framework, wodurch der korrekte und sichere Datentransfer persönlicher Daten geregelt wird. Mehr Informationen dazu finden Sie auf <a class="adsimple-311192666" href="https://www.privacyshield.gov/participant?id=a2zt0000000CbahAAC&amp;status=Active" target="_blank" rel="follow noopener noreferrer">https://www.privacyshield.gov/participant?id=a2zt0000000CbahAAC&amp;status=Active</a>.<br />
+Mehr Informationen zum Datenschutz bei StackPath bzw. BootstrapCDN finden Sie auf <a class="adsimple-311192666" href="https://www.bootstrapcdn.com/privacy-policy/?tid=311192666" target="_blank" rel="noopener">https://www.bootstrapcdn.com/privacy-policy/</a>.</p>
```

Comment:
It doesn't seem like we are using any CDN, everything is served from our server. We can definetly keep a CDN as an option for the future though, just make sure to avoid shit like google